src/ape/api/transactions.py

```diff
@@ -52,10 +52,8 @@ def total_transfer_value(self) -> int:
         Useful for determining if an account balance can afford
         to submit the transaction.
         """
-        if self.max_fee is None:
-            raise TransactionError(message="Max fee must not be null.")
-
-        return self.value + self.max_fee
+        max_fee = self.max_fee or 0
+        return self.value + max_fee
```

Comment:
This should be `self.max_fee` or `self.max_priority_fee + self.provider.base_fee`

Although, I think this might be too much detail for `TransactionAPI` and it should probably leave this open as an API method to fill in based on the structure of the transaction (whether it used base_fee) A new issue?

Reply:
I can undo these changes, I think I meant for this to be somewhere else